portability-core/src/main/java/org/dataportabilityproject/job/JobDao.java

```diff
@@ -193,96 +177,44 @@ public void updateJobStateToAssigneWithAuthData(String id, String encryptedExpor
     PortabilityJob updatedJob = existingJob.toBuilder()
         .setEncryptedExportAuthData(encryptedExportAuthData)
         .setEncryptedImportAuthData(encryptedImportAuthData)
+        .setJobState(JobState.ASSIGNED_WITH_AUTH_DATA)
         .build();
-    updateJobState(updatedJob, JobState.ASSIGNED_WITHOUT_AUTH_DATA,
-        JobState.ASSIGNED_WITH_AUTH_DATA);
+    updateJobState(updatedJob, JobState.ASSIGNED_WITHOUT_AUTH_DATA);
   }
 
   /**
    * Deletes an existing {@code job} in storage with the given {@code jobState}.
    */
-  public void deleteJob(String id, JobState jobState) throws IOException {
-    String key = createKey(jobState, id);
-    Map<String, Object> existing = storage.get(key);
-    Preconditions.checkArgument(existing != null, "Job not found");
-    storage.delete(key);
-  }
-
-  /**
-   * Updates an existing {@code job} in storage with the given {@code jobState}.
-   */
-  private void updateJob(PortabilityJob job, JobState jobState) throws IOException {
-    String key = createKey(jobState, job.id());
-    Map<String, Object> existing = storage.get(key);
-    Preconditions.checkArgument(existing != null, "Job not found");
-    // Store the updated job info
-    Map<String, Object> data = job.asMap();
-    logger.debug("Data: {}", data);
-    storage.put(key, data);
+  public void deleteJob(String jobId) throws IOException {
```

Comment:
Now that it's one line, maybe we can move this to WorkerJobDao or just inline it where needed